modular_skyrat/modules/goofsec/code/projectiles.dm

```diff
@@ -155,7 +154,7 @@
 	fire_sound = 'modular_skyrat/modules/sec_haul/sound/sfrifle_fire.ogg'
 
 /obj/projectile/beam/laser/hardlight_bullet/pitbull
-	damage = 27
+	damage = 30 // 11/20/21: Increased from 27 to 30 to improve underperforming guns.
```

Comment:
this buffs the Pitbull to a time to down of around 3.5s above its current of 5s (equal to a bulldog with buckshot, at about 2 tiles)